systemtests/tests/webui-selenium/testrunner.in

```diff
@@ -36,12 +36,25 @@ END_OF_DATA
 
 run_bareos
 
-@PHP@ -S 127.0.0.1:@php_port@ -t @PROJECT_SOURCE_DIR@/../webui/public/ >$tmp/php.out 2>&1 &
+@PHP@ -S 127.0.0.1:@php_port@ -t @PROJECT_SOURCE_DIR@/../webui/public/ >"$tmp/php.out" 2>&1 &
 
 PHP_PID=$(echo $!)
 
 # run tests but stop the test run on the first error or failure
-@PYTHON@ @CMAKE_SOURCE_DIR@/webui/tests/selenium/webui-selenium-test.py --fail >$tmp/selenium.out 2>&1
+
+#
+# test with admin profile resource
+#
+export BAREOS_WEBUI_PROFILE=admin
+@PYTHON@ @CMAKE_SOURCE_DIR@/webui/tests/selenium/webui-selenium-test.py --fail >"$tmp/selenium.out" 2>&1
+
+#
+# test with readonly profile resource
+#
+export BAREOS_WEBUI_USERNAME=ro
+export BAREOS_WEBUI_PASSWORD=ro
+export BAREOS_WEBUI_PROFILE=readonly
+@PYTHON@ @CMAKE_SOURCE_DIR@/webui/tests/selenium/webui-selenium-test.py --fail >>"$tmp/selenium.out" 2>&1
```

Comment:
You are running the same test twice. How it gets visible for the user, if the normal or the readonly test have failed? Wouldn't it be better to have separate (inherited) tests?

You could use follwoing to determine the profile automatically:
```
export BAREOS_WEBUI_PROFILE=$(echo "show console=admin" | bin/bconsole | sed -rn 's/ *Profile = "(.*)"/\1/p')
```

Reply:
> You are running the same test twice.
Yes, I do. With different parameters and as the commit message states different behaviour controlled by the added conditionals.
> How it gets visible for the user, if the normal or the readonly test have failed?
There is a logfile.
> Wouldn't it be better to have separate (inherited) tests?
Why not use the same tests?